stdlib/public/core/Reverse.swift

```diff
@@ -253,6 +253,19 @@ extension ReversedCollection: BidirectionalCollection {
 
 extension ReversedCollection: RandomAccessCollection where Base: RandomAccessCollection { }
 
+extension ReversedCollection {
+  /// This is optimization to return identity of doubly reversed collection
```

Comment:
This part of the comment is an implementation detail, I don't think it should be a doc-comment.